src/main/java/com/redhat/red/build/koji/model/xmlrpc/KojiBuildTypeQuery.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc. (jcasey@redhat.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.red.build.koji.model.xmlrpc;
+
+import org.commonjava.rwx.binding.anno.DataKey;
+import org.commonjava.rwx.binding.anno.StructPart;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@StructPart
+public class KojiBuildTypeQuery
+        extends KojiQuery
+{
+    @DataKey( "query" )
```

Comment:
So, is this supposed to create a nested struct? It seems like you're going for something like:

```
<value>
  <struct>
    <member>
      <name>query</name>
      <value>
        <struct>
          <member>
            <name>name</name>
            <value>foo</value>
          </member>
        </struct>
      </value>
    </member>
  </struct>
</value>
```

If so, what other members might be present as siblings of 'query'? If there are never any sibling members, maybe we can make a @Converter that can handle this mess, and arrive at actual fields, instead of reverting to python programming philosophy?